iree/hal/vulkan/vulkan_driver.cc

```diff
@@ -193,19 +205,22 @@ StatusOr<ref_ptr<VulkanDriver>> VulkanDriver::CreateUsingInstance(
 
   return assign_ref(new VulkanDriver(
       std::move(syms), instance, /*owns_instance=*/false,
-      std::move(debug_reporter), std::move(options.device_extensibility)));
+      std::move(debug_reporter), std::move(options.device_extensibility),
+      /*renderdoc_capture_manager=*/nullptr));
```

Comment:
This is clunky.

* `CreateUsingInstance` can't *load* RenderDoc since it would be too late with an existing Instance
  * I'm not sure if it's worth plumbing RenderDoc (or generic capture recorder) integration up through the C API to allow users to control capture recording or bring their own recorder.
* In the case where RenderDoc is already loaded (launched from its UI), we *could* connect and still add extra information via `SetCaptureFileComments`, trigger explicit captures, etc.

Reply:
I think we can avoid putting it in the API for now. In theory an integrating app that is creating the vkinstance would be the one responsible for setting up renderdoc and performing the captures.

Reply:
Okay, I added a comment around here with a bit of context.